partner_phone_secondary/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 - Iván Todorovich
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    phone2 = fields.Char("Phone (Secondary)")
+
+    @api.onchange("phone2", "country_id", "company_id")
+    def _onchange_phone2_validation(self):
```

Comment:
I think the filed should be turned into computed readonly False one.

@emagdalenaC2i

Reply:
@rousseldenis Isn't readonly false by default?

Reply:
Computed fields are readonly=True by default.

But you can declare a field as computed, readonly=False. That was developed in order to replace completely the onchanges.

But, to be tested if it allows the same behaviour (format the field that triggered the compute)